### DISCUSSION TITLE
I can't renew my pack

### DISCUSSION THREAD

**Discussion by gabsdemarco** [community/community#56803] [GitHub Education]
### Hi everyone,

Hi, I haven't been able to renew Student Developer Pack. Please, can anyone help? Is there any email address I can use to request some support?
Thanks !

![image](https://github.com/community/community/assets/101835241/bf2c71ad-6946-4a4e-96ce-6a85bc8b1e4d)

**Comment by samus-aran** [comment]
Hi @gabsdemarco 👋 

Have you been asked to reverify? Did you receive a timescale when your verification needed to be provided?
